environment_setup/install_cubemx.sh

```diff
@@ -46,9 +46,8 @@ else
     echo -n $AUTO_INSTALL_XML > auto-install.xml
 
     # create script to cd and run STM32CubeMX
-    STM32_SCRIPT="pushd /opt/STM32CubeMX_$CUBE_VERSION && ./STM32CubeMX && popd"
-
-    echo -n "$STM32_SCRIPT" > cuberunner.sh
+    echo -n $'#!/bin/bash \n\n ' > cuberunner.sh
+    echo -n "pushd /opt/STM32CubeMX_$CUBE_VERSION && ./STM32CubeMX && popd" >> cuberunner.sh
```

Comment:
Can we have a comment explaining what going on?

Reply:
Happy to add a comment, what would help here?

Reply:
To start with, why are we  redirecting '#!/bin/bash \n\n ' to a script?

Reply:
Because we need `#!/bin/bash` at the start of the script, with a newline after it. The second newline is just for readability once the script is generated.